lib/string_tools.rb

```diff
@@ -226,6 +226,7 @@ def remove_links(node, attr_name)
     end
 
     class IframeNormalizer
+      HOSTING_REG = %r{^http(?:s?):\/\/?(:www\.)?rutube\.ru(\/video|\/play\/embed)|youtu(?:be\.com\/(watch|shorts|embed)|\.be\/|be\-nocookie)}.freeze
```

Comment:
[Correctable] Style/RedundantRegexpEscape: Redundant escape inside regexp literal<br>Layout/LineLength: Line is too long. [149/120]